.travis/install_gradle.sh

```diff
@@ -1,10 +1,8 @@
-#!/bin/bash
-set -e
-set -x
+#!/bin/bash -ex
 
 pushd /tmp
 
-wget http://services.gradle.org/distributions/gradle-2.4-all.zip
+wget $(wget -qO- https://services.gradle.org/versions/current | jq -r .downloadUrl)
```

Comment:
This obfuscates what's getting pulled, and means that a build will break if something external to the project is updated.

If you'd like us to maintain compatibility with "latest gradle" then perhaps this is a separate pull request; but I get the impression this PR is to "clean things up", and I don't agree that obfuscating the version we're supporting actually helps in that respect.